src/urllib3/contrib/ntlmpool.py

```diff
@@ -5,13 +5,25 @@
 """
 from __future__ import absolute_import
 
+import warnings
 from logging import getLogger
 
 from ntlm import ntlm
 
 from .. import HTTPSConnectionPool
 from ..packages.six.moves.http_client import HTTPSConnection
 
+
+warnings.warn(
+    "The 'urllib3.contrib.nltmpool' module is deprecated and will be removed "
+    "in a later urllib3 v2.0 release for a possible third-party library as "
+    "urllib3 is not able to support it properly due to reasons listed in the "
+    "issue: https://github.com/urllib3/urllib3/issues/2282, If you are a user "
+    "of this module please have your say in the same issue above.",
+    DeprecationWarning,
+    stacklevel=2,
```

Comment:
Remove the `stacklevel=2`. We're already in the global scope so no need to make the warning appear to have come from a different level in the call stack.